doc/tutorials/acrn_on_qemu.rst

```diff
@@ -186,10 +188,13 @@ Install ACRN Hypervisor
          insmod ext2
 
          echo 'Loading ACRN hypervisor ...'
-         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out  root=/dev/vda5
          module /boot/bzImage Linux_bzImage
       }
 
+     .. note::
+     If your rootfs partition isn't vda5, please change it to match with yours.
+
 #. Update GRUB:
 
    .. code-block:: none
```

Comment:
At line 259, what script is the user updating? Is it OVMF.fd? Why OVMF.fd instead of a launch script *.sh?

Reply:
> At line 259, what script is the user updating? Is it OVMF.fd? Why OVMF.fd instead of a launch script *.sh?

@amyreye 
259 #. Update the script to use your disk image (``UserVM.img`` or ``UserVM.iso``). 
It isn't OVMF.fd; but the path of your User VM image.

Reply:
What is the name of the script?

Reply:
> What is the name of the script?

The whole content of the script was pasted in this doc, user can name it to what they want.